content/en/docs/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -214,9 +236,40 @@ Check the pod `authservice-0` is in running state in namespace `istio-system`:
 kubectl get pod authservice-0 -n istio-system
 ```
 
-## Next steps
+### Extra network setup requirement for `vpc-gen2` clusters only
+
+**Note**: These steps are not required for Classic clusters, i.e. where `WORKER_NODE_PROVIDER` is set to `classic`.
```

Comment:
```suggestion
**Note**: These steps are not required for `classic` clusters, i.e. where `WORKER_NODE_PROVIDER` is set to `classic`.
```

Since the beginning, I think you refer to it as `classic`, so to avoid confusing the users, maybe we should stick to `classic` instead of "Classic".